NOTICE-binary

```diff
@@ -205,7 +213,7 @@ and decompression library written by Adrien Grand. It can be obtained at:
   * LICENSE:
     * license/LICENSE.lz4.txt (Apache License 2.0)
   * HOMEPAGE:
-    * https://github.com/yawkat/lz4-java
+    * https://github.com/jpountz/lz4-java
```

Comment:
Hi, @LuciferYang . I recovered this becauseI realized that this is a part of Netty's NOTICE file instead of ours.

https://github.com/netty/netty/blob/511cbacea64c3348078948e8cb424749afeaeefc/NOTICE.txt#L118